tutorials/dev-app-cli.adoc

```diff
@@ -0,0 +1,128 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="dev-app-cli"]
+= Tutorial: Deploying an application by using the CLI
+include::_attributes/common-attributes.adoc[]
+:context: dev-app-cli
+
+toc::[]
+
+This tutorial walks you through deploying services to stand up an application called `national-parks-app` on {product-title} that displays a map of national parks across the world. You will use the {oc-first} to complete this tutorial.
+
+To complete this tutorial, you will perform the following steps:
+
+. xref:../tutorials/dev-app-cli.adoc#getting-started-cli-creating-new-project_dev-app-cli[Create a new project for the application].
++
+This step allows your application to be isolated from other cluster user's workloads.
+
+. xref:../tutorials/dev-app-cli.adoc#getting-started-cli-granting-permissions_dev-app-cli[Grant view permissions].
++
+This step grants view permissions to help discover services and other resources running within the project.
```

Comment:
Maybe something like:

This step grants view permissions to properly interact with the OpenShift AI, thus helping to discover services and other resources running within the project.